### DISCUSSION TITLE
Refactoring & onboarding

### DISCUSSION THREAD

**Discussion by jsuchal** [slovensko-digital/priznanie-digital#783] [General]
V nadvaznosti na mensi flamewar :fire: zo zaciatku roka opatovne chceme otvorit temu refactoru kodu priznanie.digital a onboardingu ludi do projektu.

Pointa:
- chceme aby bol vyvoj co najrychlejsi + onboarding sa pre novych ludi zjednodusil.

Navrh od @idpsycho je organizacia kodu podla features.
- povodny PR a nejake diskusia tam https://github.com/slovensko-digital/priznanie-digital/pull/685 povodny refactor **precitajte si to predtym, ako budete sem pisat**

Kedze za sebou mame release s uplne novymi ludmi, bude dobre zachytit ich blockers/skusenosti pri onboardingu do projektu. Nelimitujme sa tu teda len uplne len na technicku cast onboardingu.

Zaroven ludia co do kodu hlbkovo vidia - napr. @julius-retzer, @MarekVigas nech spisu slabe miesta v kode, ktore by stali za upravu/refactor.

Diky za vsetky postrehy.

PS. Skusme civilizovane, skor ako nieco napisete, 3x nadych/vydych, 10 drepov, precitajte si to po sebe, prechadzka po byte a potom odoslat ak to sami nezmazete.

**Comment by MarekVigas** [comment]
ping @idpsycho 

teraz mame priestor a cas to doriesit
